packages/contracts/src/contracts/current/test/Mintable/Mintable.sol

```diff
@@ -29,7 +30,10 @@ contract Mintable is UnlimitedAllowanceToken, SafeMath {
     function mint(uint256 _value)
         public
     {
-        require(_value <= 100000000000000000000);
+        require(
+            _value <= 100000000000000000000,
+            "Minting more than 100000000000000000000 is not allowed."
```

Comment:
Should we make these error messages constants too?

Reply:
I guess since they are only used in a single please, I'm personally ok with leaving them like this.

Reply:
I'm fine leaving this as is for test/example contracts.